config/jobs/containerd/cri/containerd-cri-presubmit-jobs.yaml

```diff
@@ -85,7 +85,7 @@ presubmits:
     spec:
       containers:
       - name: pull-cri-containerd-node-e2e
-        image: gcr.io/k8s-testimages/kubekins-e2e:v20191127-ee0721a-master
+        image: gcr.io/k8s-testimages/kubekins-e2e:v20191127-ee0721a-1.15
```

Comment:
This job tests against the master branch in k8s; why use a 1.15 image?

Reply:
It is testing against 1.15m right?
```
 - org: kubernetes
      repo: kubernetes
      base_ref: release-1.15
```

Reply:
You're right. 

I think the fact that all the jobs are named "pull-cri-containerd-node-e2e" confused me....